<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoSerial</class>
 <widget class="QMainWindow" name="LoSerial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoSerial</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0,1">
      <item>
       <widget class="QGroupBox" name="COMSetting">
        <property name="styleSheet">
         <string notr="true">QComboBox { width: 65px; }</string>
        </property>
        <property name="title">
         <string>COM Setting</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>COM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="COMs">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="currentText">
              <string notr="true"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Baud Rate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="BaudRate">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="currentText">
              <string notr="true">110</string>
             </property>
             <item>
              <property name="text">
               <string>110</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>300</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4800</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">19200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">38400</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">43000</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">56000</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">57600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">115200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>250000</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1000000</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Data Bits</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="DataBits">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="currentText">
              <string notr="true">8</string>
             </property>
             <item>
              <property name="text">
               <string notr="true">8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">5</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Stop Bits</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="StopBits">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="currentText">
              <string notr="true">1</string>
             </property>
             <item>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">1.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string notr="true">2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Parity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="ParityMode">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Even</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Odd</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Space</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Mark</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Flow Control</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="FlowControl">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hardware</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Software</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="RxSetting">
        <property name="title">
         <string>Rx Setting</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="HexRx">
           <property name="text">
            <string>HEX</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="TxSetting">
        <property name="title">
         <string>Tx Setting</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QCheckBox" name="HexTx">
               <property name="text">
                <string>HEX</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Echo">
               <property name="text">
                <string>Echo</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="AttachCR">
               <property name="text">
                <string>Attach CR</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="AutoClear">
               <property name="text">
                <string>Aut Clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,0,1">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QCheckBox" name="AutoTx">
             <property name="text">
              <string>Aut Send /</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="AutoTxInterval">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>65535</number>
             </property>
             <property name="value">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string> ms</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="Open">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Clear">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>211</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
      <item>
       <widget class="QTextEdit" name="RxData">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TxData"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionChinese"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Clear</sender>
   <signal>clicked()</signal>
   <receiver>RxData</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AutoTx</sender>
   <signal>toggled(bool)</signal>
   <receiver>AutoTxInterval</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
